### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by kopelli** [SlavaVedernikov/C4InterFlow#97] [Feedback]
I'm finally getting some time to dig into C4InterFlow after creating my fork months ago. But something I'm noticing is that there's not really any guidance as a developer to getting started. Also noticing some full, hardcoded paths (like in the [launchSettings.json](https://github.com/SlavaVedernikov/C4InterFlow/blob/863aa8adf76e760f70367e169f716a96fee9b399/C4InterFlow.Cli/Properties/launchSettings.json#L5)) that make it difficult to just clone the repo and run it...at least from Visual Studio. Using things like well-known MSBuild properties/environment variables.

Also things like code convention preferred in the project so PR's are more about intent/features rather than code consistency. Stuff like .editorconfig/ReSharper shared project settings/etc.

Considering you've been solo on this for so long, would you be open to those sorts of PRs?

I have a bigger need for documenting my company's deployment infrastructure, so I wanted to see if I could start implementing Issue #62 but ran into those difficulties so far.

**Comment by kopelli** [comment]
@SlavaVedernikov in case you haven't seen this yet.

**Reply by SlavaVedernikov** [reply]
Hi @kopelli, sorry I missed your message.

I would love to work with you on "guidance as a developer to getting started". As you said I have been working on this practically solo, so it was never an issue. I've added #100 for that. Please add you suggestions in comments, so we can shape it together.

I'm also happy for you to have a look into #62.
With this one I'd like to agree on JSON schema first.

Thanks for your feedback :)
